Build/AndroidDeprecationWarning.targets

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="AndroidDeprecationWarn">
+    <Error
+      Condition="'$(HCLibPlatformType)' == 'Android' OR '$(AndroidAPILevel)' != ''"
+      Text="ANNOUNCEMENT: The Visual Studio-based Android build is being removed. Please see https://github.com/microsoft/libHttpClient/issues/ISSUE_NUMBER for more details on this announcement. To use the VS build in the deprecation period, silence this error in AndroidDeprecationWarning.targets." />
```

Comment:
>https://github.com/microsoft/libHttpClient/issues/ISSUE_NUMBER [](start = 93, length = 62)

should be 

[https://github.com/microsoft/libHttpClient/issues/573](https://github.com/microsoft/libHttpClient/issues/573)